<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MyController</class>
 <widget class="QWidget" name="MyController">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>253</width>
    <height>400</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>200</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>251</width>
     <height>381</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="title">
       <string>Поворот</string>
      </property>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>11</x>
         <y>31</y>
         <width>41</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>х</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>12</x>
         <y>69</y>
         <width>41</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>y</string>
       </property>
      </widget>
      <widget class="QSlider" name="xhorizontalSlider">
       <property name="geometry">
        <rect>
         <x>31</x>
         <y>33</y>
         <width>160</width>
         <height>16</height>
        </rect>
       </property>
       <property name="maximum">
        <number>359</number>
       </property>
       <property name="sliderPosition">
        <number>180</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
      <widget class="QLineEdit" name="rot_xEdit">
       <property name="geometry">
        <rect>
         <x>194</x>
         <y>31</y>
         <width>51</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="rot_yEdit">
       <property name="geometry">
        <rect>
         <x>194</x>
         <y>70</y>
         <width>51</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
      <widget class="QSlider" name="yhorizontalSlider">
       <property name="geometry">
        <rect>
         <x>31</x>
         <y>74</y>
         <width>160</width>
         <height>16</height>
        </rect>
       </property>
       <property name="maximum">
        <number>359</number>
       </property>
       <property name="sliderPosition">
        <number>180</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
      <widget class="QLineEdit" name="rot_zEdit">
       <property name="geometry">
        <rect>
         <x>194</x>
         <y>108</y>
         <width>51</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
      <widget class="QSlider" name="zhorizontalSlider">
       <property name="geometry">
        <rect>
         <x>31</x>
         <y>111</y>
         <width>160</width>
         <height>16</height>
        </rect>
       </property>
       <property name="maximum">
        <number>359</number>
       </property>
       <property name="sliderPosition">
        <number>180</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>13</x>
         <y>106</y>
         <width>41</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>z</string>
       </property>
      </widget>
      <widget class="QPushButton" name="rotateButton">
       <property name="geometry">
        <rect>
         <x>167</x>
         <y>139</y>
         <width>81</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Повернуть</string>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>160</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="title">
       <string>Масштабирование        Перенос</string>
      </property>
      <widget class="QLineEdit" name="kxEdit">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>30</y>
         <width>61</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>90</y>
         <width>41</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>kz = </string>
       </property>
      </widget>
      <widget class="QPushButton" name="scaleButton">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>120</y>
         <width>121</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Масштабировать</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="kyEdit">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>60</y>
         <width>61</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="kzEdit">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>90</y>
         <width>61</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_12">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>41</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>kx = </string>
       </property>
      </widget>
      <widget class="QLabel" name="label_13">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>60</y>
         <width>41</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>ky = </string>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>90</y>
         <width>41</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>dz = </string>
       </property>
      </widget>
      <widget class="QLabel" name="label_14">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>30</y>
         <width>41</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>dx = </string>
       </property>
      </widget>
      <widget class="QLineEdit" name="dzEdit">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>90</y>
         <width>61</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_15">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>60</y>
         <width>41</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>dy = </string>
       </property>
      </widget>
      <widget class="QLineEdit" name="dxEdit">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>30</y>
         <width>61</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="dyEdit">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>60</y>
         <width>61</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="moveButton">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>120</y>
         <width>101</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Перенести</string>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="fileButton">
      <property name="text">
       <string>Загрузить модель</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
